ext/lb/nginx/nginx.go

```diff
@@ -44,11 +45,17 @@ func (p *NginxLoadBalancer) HandleEvent(event *dockerclient.Event) error {
 }
 
 func (p *NginxLoadBalancer) Template() string {
-	if p.cfg.NginxPlusEnabled {
-		return nginxPlusConfTemplate
+	d, err := ioutil.ReadFile(p.cfg.TemplatePath)
+
+	if err == nil {
+		return string(d)
+	} else {
+		log().Infof("Missing nginx configuration template: file=%s", p.cfg.TemplatePath)
```

Comment:
We have this spread throughout. We should not log.Fatal in the extensions. This should be updated to return `(string, error)` and then bubble the error up.